src/hpnmg/util/Logging.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#define HPNMG_LOGGING
```

Comment:
Is this supposed to be here? This defeats the `#ifdef HPNMG_LOGGING` checks in this header and its [implementation file](https://github.com/jannikhuels/hpnmg/pull/22/files#diff-770d0520268fd855c762ecc0481fe236R3).